bag_transfer/accession/views.py

```diff
@@ -78,7 +78,7 @@ def handle_ajax_request(self, request):
         return self.render_to_json_response(rdata)
 
 
-class AccessionDetailView(PageTitleMixin, AccessioningArchivistMixin, DetailView):
```

Comment:
I think we should probably use the OrgReadViewMixin here to prevent users from seeing accessions that don't belong to their organization.